src/ralph/models/xapi/fields/verbs.py

```diff
@@ -11,28 +15,42 @@
 
 
 class ViewedVerbField(BaseModelWithConfig):
-    """Represents the `verb` xAPI Field for page viewed xAPI statement.
+    """Represents the `verb` xAPI Field for the action `viewed`.
 
     Attributes:
        id (str): Consists of the value `http://id.tincanapi.com/verb/viewed`.
        display (dict): Consists of the dictionary `{"en": "viewed"}`.
     """
 
-    id: VERB_VIEWED_ID = VERB_VIEWED_ID.__args__[0]
-    display: dict[LANG_EN_DISPLAY, VERB_VIEWED_DISPLAY] = {
-        LANG_EN_DISPLAY.__args__[0]: VERB_VIEWED_DISPLAY.__args__[0]
+    id: Literal[VERB_VIEWED_ID] = VERB_VIEWED_ID
```

Comment:
I was wrong, it's possible to have variables in Literals. Although `pylance` complains about it and marks everything in red - but it works.